### DISCUSSION TITLE
How to configure "use_template" regarding PTI training for a person(images without caption)?

### DISCUSSION THREAD

**Discussion by terrificdm** [cloneofsimo/lora#215] [Q&A]
Should I configure --use_template="object"? Or for training a specific person, I should tag each image with caption and configure -use_template="none"(default)?
And another question is if I want to use final training result(result including TI and Lora together) in automatic111/webui, how can I set different ratio for unet and text_encoder? Regarding "text_encoder", is it a same thing with dreambooth 's text_encoder, or is it related with TI?
Thanks in advance!

**Comment by terrificdm** [comment]
I checked out the source codes, found that I need to use template="object" to train a person without captions for training images. But the problem is overfitting, I cannot change characters of person, like hair, with or without glasses etc. And I noticed there is a "proxy_token" which I can use as class token. My question is does this have similar meaning with class prompt in DreamBooth? And where can I configure "prior preservation loss" and class images(e.g. directory, number) in PTI?

Another question is what are "run_lorpt.sh" "train_lora_w_ti.py)" two scripts used for? They are similar to DreamBooth+Lora scripts in the same folder, when should I use those instead of use DreamBooth+Lora scripts?

Can anyone explain a little bit? Thanks.

**Reply by xdfeng07** [reply]
it seems that the run_lorpt.sh and train_lora_w_ti.py is obsolete or something??
